lib/vuln/http_cors/engine.py

```diff
@@ -75,6 +75,8 @@ def http_cors(target, port, timeout_sec, log_in_file, language, time_sleep,
             req = requests.get(target, headers=headers)
             if req.headers['Access-Control-Allow-Origin'] == "*":
                 return True
+            elif req.headers['Access-Control-Allow-Origin'] == "http://example.foo" and req.headers['Access-Control-Allow-Credentials'] == "true":
```

Comment:
Why in the actual reality would you check for `http://example.foo`? Checking for CORS misconfiguration involves checking for the following:
- wildcard value (implemented)
- third party allowance (not only `http://example.foo` lol)
- origin reflection
- invalid value in ACAO header
- post-domain wildcard
- pre-domain wildcard
- null-origin allowance
- http-origin allowance
- unescaped regular expressions

Reply:
Thank you for the suggestion. I added all the factors.